assets/src/blocks/Media/MediaEditor.js

```diff
@@ -1,76 +1,74 @@
-import { Fragment, useCallback } from "@wordpress/element";
-import { PanelBody } from '@wordpress/components';
+import { Fragment, useCallback } from '@wordpress/element';
+import { PanelBody, TextControl } from '@wordpress/components';
 import { MediaPlaceholder, InspectorControls } from '@wordpress/block-editor';
-import { TextControl } from '@wordpress/components';
 import { debounce } from 'lodash';
 
 import { MediaElementVideo } from './MediaElementVideo';
 import { useSelect } from '@wordpress/data';
 import { lacksAttributes } from './MediaBlock';
 
 const { __ } = wp.i18n;
-const {apiFetch} = wp;
-const {addQueryArgs} = wp.url;
+const { apiFetch } = wp;
+const { addQueryArgs } = wp.url;
 const { RichText } = wp.blockEditor;
 
-
-const MediaInspectorOptions = ({ attributes, setAttributes }) => {
+const MediaInspectorOptions = ( { attributes, setAttributes } ) => {
   const { media_url } = attributes;
 
-  const updateEmbed = async media_url => {
+  const updateEmbed = async ( mediaUrl ) => {
     let embed_html;
     try {
-      const embedPreview = await apiFetch({
-        path: addQueryArgs('/oembed/1.0/proxy', { url: resolveURL(media_url) }),
-      });
+      const embedPreview = await apiFetch( {
+        path: addQueryArgs( '/oembed/1.0/proxy', { url: resolveURL( mediaUrl ) } ),
+      } );
 
       embed_html = embedPreview ? embedPreview.html : null;
-    } catch (error) {
+    } catch ( error ) {
       embed_html = null;
     }
-    setAttributes({ media_url, embed_html });
+    setAttributes( { mediaUrl, embed_html } );
```

Comment:
I think here it should be `setAttributes( { media_url: mediaUrl, embed_html } );`

Reply:
Done.